hbase/Dockerfile

```diff
@@ -104,6 +106,21 @@ RUN export LIBS=$(find /stackable/hadoop/share/hadoop -name '*.jar' -printf '%p:
 
 ENV HBASE_CONF_DIR=/stackable/hbase/conf
 
+# HBase 2.6 needs these dependencies to configure log4j2 with XMLMapper as output format
+# Previous versions do not need this and the argument is not set in conf.py
+RUN if [[ -n "$JACKSON_DATAFORMAT_XML" ]]; then \
+    curl -L --fail -o /stackable/hbase-${PRODUCT}/lib/jackson-dataformat-xml-${JACKSON_DATAFORMAT_XML}.jar \
+    https://repo.stackable.tech/repository/packages/jackson-dataformat-xml/jackson-dataformat-xml-${JACKSON_DATAFORMAT_XML}.jar; \
+    fi
+
+# ===
+# OPA Authorizer
+RUN if [[ -n "$OPA_AUTHORIZER" ]]; then \
+  git clone --depth 1 --branch "$OPA_AUTHORIZER" https://github.com/stackabletech/hbase-opa-authorizer.git && \
+  mvn -DskipTests package -fhbase-opa-authorizer  && \
+  cp target/hbase-opa-authorizer*.jar /stackable/hbase-${PRODUCT}/lib; \
+  fi
```

Comment:
```suggestion
RUN if [[ -n "$JACKSON_DATAFORMAT_XML" ]]; then \
      curl -L --fail -o /stackable/hbase-${PRODUCT}/lib/jackson-dataformat-xml-${JACKSON_DATAFORMAT_XML}.jar \
      https://repo.stackable.tech/repository/packages/jackson-dataformat-xml/jackson-dataformat-xml-${JACKSON_DATAFORMAT_XML}.jar; \
    fi

# ===
# OPA Authorizer
RUN if [[ -n "$OPA_AUTHORIZER" ]]; then \
      git clone --depth 1 --branch "$OPA_AUTHORIZER" https://github.com/stackabletech/hbase-opa-authorizer.git && \
      mvn -DskipTests package -fhbase-opa-authorizer  && \
      cp target/hbase-opa-authorizer*.jar /stackable/hbase-${PRODUCT}/lib; \
    fi
```

Reply:
applied a variant of this: https://github.com/stackabletech/docker-images/pull/701/commits/23b3ec317c4a7f101023aa2b46d33530f9e75b33